Automattic/wp-calypso
# Add data flow for email open stats country view

#### Proposed Changes

* This PR adds all transformations of country data for the email open stats view

#### Testing Instructions

* Visit `/stats/email/open/<site>/day/<postid>?flags=newsletter/stats&startDate=2022-12-28` and look at the Redux store using Redux Devtools.

The shape of days with email opens should be like this:

stats["emails"][siteId][postId][period]["opens"] 

<img width="710" alt="CleanShot 2022-12-29 at 15 09 01@2x" src="https://user-images.githubusercontent.com/528287/209965091-b5dd4364-42c0-488c-86fa-923472a16900.png">

Inside the countries key you should see all necessary data.


#### Pre-merge Checklist



- [ X] Has the general commit checklist been followed? (PCYsg-hS-p2)


Related to #
